<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CopyPasteDialog</class>
 <widget class="QDialog" name="CopyPasteDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>555</width>
    <height>362</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Clipboard</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>World</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QListWidget" name="worldCat">
         <property name="maximumSize">
          <size>
           <width>180</width>
           <height>16777215</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>Property Definitions</string>
          </property>
          <property name="checkState">
           <enum>Checked</enum>
          </property>
          <property name="flags">
           <set>ItemIsSelectable|ItemIsUserCheckable|ItemIsEnabled</set>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Templates</string>
          </property>
          <property name="checkState">
           <enum>Checked</enum>
          </property>
          <property name="flags">
           <set>ItemIsSelectable|ItemIsUserCheckable|ItemIsEnabled</set>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Object Types</string>
          </property>
          <property name="checkState">
           <enum>Checked</enum>
          </property>
          <property name="flags">
           <set>ItemIsSelectable|ItemIsUserCheckable|ItemIsEnabled</set>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Object Groups</string>
          </property>
          <property name="checkState">
           <enum>Checked</enum>
          </property>
          <property name="flags">
           <set>ItemIsSelectable|ItemIsUserCheckable|ItemIsEnabled</set>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QStackedWidget" name="stackedWidget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="pageList">
          <layout class="QVBoxLayout" name="verticalLayout">
           <property name="margin">
            <number>0</number>
           </property>
           <item>
            <widget class="QTreeWidget" name="worldTree">
             <property name="rootIsDecorated">
              <bool>false</bool>
             </property>
             <property name="headerHidden">
              <bool>true</bool>
             </property>
             <property name="columnCount">
              <number>2</number>
             </property>
             <column>
              <property name="text">
               <string notr="true">1</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string notr="true">2</string>
              </property>
             </column>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="widget" native="true">
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <property name="margin">
               <number>0</number>
              </property>
              <item>
               <widget class="QToolButton" name="worldCheckAll">
                <property name="text">
                 <string>Check All</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QToolButton" name="worldCheckNone">
                <property name="text">
                 <string>Check None</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Cells</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QListWidget" name="cellCat">
         <property name="maximumSize">
          <size>
           <width>180</width>
           <height>16777215</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>Properties</string>
          </property>
          <property name="checkState">
           <enum>Checked</enum>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Templates</string>
          </property>
          <property name="checkState">
           <enum>Checked</enum>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Lots</string>
          </property>
          <property name="checkState">
           <enum>Checked</enum>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Objects</string>
          </property>
          <property name="checkState">
           <enum>Checked</enum>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Map</string>
          </property>
          <property name="checkState">
           <enum>Checked</enum>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QStackedWidget" name="stackedWidget_2">
         <widget class="QWidget" name="page">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <property name="margin">
            <number>0</number>
           </property>
           <item>
            <widget class="QTreeWidget" name="cellTree">
             <property name="headerHidden">
              <bool>true</bool>
             </property>
             <property name="columnCount">
              <number>2</number>
             </property>
             <column>
              <property name="text">
               <string notr="true">1</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string notr="true">2</string>
              </property>
             </column>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="widget_2" native="true">
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <property name="margin">
               <number>0</number>
              </property>
              <item>
               <widget class="QToolButton" name="cellCheckAll">
                <property name="text">
                 <string>Check All</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QToolButton" name="cellCheckNone">
                <property name="text">
                 <string>Check None</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Options</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Copying</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QRadioButton" name="copyReplaceRadio">
            <property name="text">
             <string>Replace clipboard contents with stuff being copied</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="copyMergeRadio">
            <property name="text">
             <string>Merge clipboard contents with stuff being copied</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Pasting</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QRadioButton" name="pasteReplaceRadio">
            <property name="text">
             <string>Replace cell contents with stuff being pasted</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="pasteMergeRadio">
            <property name="text">
             <string>Merge cell contents with stuff being pasted</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CopyPasteDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CopyPasteDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
